filament/include/filament/Box.h

```diff
@@ -21,10 +21,14 @@
 
 #include <utils/compiler.h>
 
-#include <limits>
-
+#include <math/mat3.h>
 #include <math/mat4.h>
 #include <math/vec3.h>
+#include <math/vec4.h>
+
+#include <float.h>
+
+#include <stddef.h>
```

Comment:
Does Filament prefer c-style headers over c++ (i.e. <cstddef> as a convention?

Reply:
Yes. I'm trying to avoid the use of the STL in public headers as much as possible, so that we don't create unnecessary dependencies and reduce compilation times.